File: BookieApp/BookieApp/ModelUser/RegisterRequest.swift
//
//  RegisterRequest.swift
//  BookieApp
//
//  Created by dam2 on 4/4/24.
//

import Foundation

struct RegisterRequest: Encodable, Decodable{
    
    var rol: String = "ROLE_USER"
    var nombre: String
    var username: String
    var password: String
    var email: String
    var ciudad: String
    var provincia: String
    var codigoPostal: String
    var foto: String
    var reportado: Bool
    var token: String
}

File: estechbackscaffolding-2/src/main/java/com/example/securityscaffolding/dto/RegisterRequest.java
package com.example.securityscaffolding.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@AllArgsConstructor
@NoArgsConstructor
@Data
@Builder
public class RegisterRequest {

    private String rol;
    private String nombre;
    private String username;
    private String password;
    private String email;
    private String ciudad;
    private String provincia;
    private Integer codigoPostal;
    private String foto;
    private Boolean reportado;
    private String token;
    private Integer bookieFavoritaId;
}
